content/en/docs/armory-admin/evaluate-artifacts-stage-enable.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Enable the Evaluate Artifacts Stage Plugin
+description: >
+  Enable the Evaluate Artifacts Stage plugin to add a stage to Armory Enterprise that makes it easy to evaluate SpEL expressions inside of Spinnaker™ artifacts.
+aliases:
+  - /docs/plugin-guide/plugin-evaluate-artifacts/
+---
+
+![Proprietary](/images/proprietary.svg)
+
+## Overview
+
+Armory’s Evaluate Artifacts plugin allows you to easily evaluate SpEL queries inside of any Spinnaker artifacts. Some artifact types and their associated stages, such as Kubernetes, support  using SpEL to read parameters from the Spinnaker context and inject them into an artifacts manifest at deploy time. Other stages, such as the Terraform Integration stage, do not support this out of the box. The Evaluate Artifacts Stage plugin adds supports for evaluating SpEL against any artifact within your Spinnaker pipeline.
+
+For information about how to use the stage, see [Use the Evaluate Artifacts Stage]({{< ref "evaluate-artifacts-plugin-use.md" >}}).
+
+## Setup
+
+Add the following snippet to your Operator config, such as `spinnakerservice.yml`:
+
+{{< prism lang=yaml line="4-5, 10, 22" >}}
+    spec:
+      spinnakerConfig:
+        profiles:
+          spinnaker:  
+            spinnaker:
+              extensibility:
+                plugins:
+                  Armory.EvaluateArtifactsPlugin:
+                    enabled: true
+                    version: <PLUGIN_VERSION> # Replace with the version you want to use
+              repositories:
+                evaluateArtifacts:
+                  url: https://raw.githubusercontent.com/armory-plugins/evaluate-artifacts-releases/master/repositories.json
+          gate:
+            spinnaker:
+              extensibility:
+                deck-proxy:
+                  enabled: true
+                  plugins:
+                    Armory.EvaluateArtifactsPlugin:
+                      enabled: true
+                      version: <PLUGIN_VERSION> # Replace with the version you want to use. Omit the 'v' when providing a version. For example, use 0.1.0, not v0.1.0
+{{< /prism >}}
+
+Keep the following in mind when using this configuration snippet: 
+
+* Make sure to include the nested `spinnaker` parameters on lines 4 and 5. Both are required because of how Operator configs are processed and consumed by Armory Enterprise.
+* Replace `<PLUGIN_VERSION>` on lines 10 and 22 with the version of the plugin that you want to use. Plugin versions can be found [here](#versions).
+
+Then, deploy your updated Operator configuration using one of the following methods:
+
+- The `deploy.sh` script if you use the [Armory kustomize repo](https://github.com/armory/spinnaker-kustomize-patches) to help manage your Operator configurations
+- `kubectl -n <spinnaker-namespace> -f <operator-config>.yml​`
```

Comment:
You are deploying an Armory Enterprise manifest, not an updated Operator configuration. 

My understanding of `deploy.sh` is that it's for a quickstart deployment only, that you wouldn't use it for deploying an updated Armory Enterprise manifest. How you deploy Spinnaker or AE depends on whether you are using a single `spinnakerservice.yml` manifest file or are using Kustomize patches.  

If using a single manifest file:
```bash
kubectl -n <namespace> apply -f <path-to-manifest-file>
```

If using Kustomize patches:
```bash
kubctl -n <namespace> apply -k <path-to-kustomize-directory>
```
or if you have Kustomize installed:

```bash
kustomize build | kubectl -n <namespace> apply -f -
```